release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2815,6 +2815,8 @@ This update contains changes from Kubernetes 1.22.6 up to 1.22.8. More informati
 
 * Previously, the *Git import* page failed to load due to performance improvement releases. With this fix, the *Git import* page no longer fails to load and now runs as intended. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069621#[*BZ#2069621*])
 
+* Before this update, if users provided a secret for a private repository in the *Import from Git* form, the secret was not decoded. As a result, the *Import from Git* form failed to detect the correct import strategy and builder image for the repository.  With this update, the *Import from Git* form decodes the secret before its use, which helps detect the correct import strategy and builder image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069258[*BZ#2069258*])
```

Comment:
```suggestion
* Before this update, if users provided a secret for a private repository in the *Import from Git* form, the secret was not decoded. As a result, the *Import from Git* form failed to detect the correct import strategy and builder image for the repository. With this update, the *Import from Git* form decodes the secret before its use, which helps detect the correct import strategy and builder image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069258[*BZ#2069258*])
```
Remove the extra space between the 2nd and 3rd sentences

Reply:
Incorporated